samples/bot-ai-meeting-helper/package.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "aimeetinghelper",
+    "version": "1.0.0",
+    "msteams": {
+        "teamsAppId": null
+    },
+    "description": "Microsoft Teams Toolkit AI Chat Bot Sample with Teams AI Library",
+    "engines": {
+        "node": "16 || 18"
+    },
+    "author": "Microsoft",
+    "license": "MIT",
+    "main": "./src/index.js",
+    "scripts": {
+        "dev:teamsfx": "env-cmd --silent -f .localConfigs npm run dev",
+        "dev:teamsfx:testtool": "env-cmd --silent -f .localConfigs.testTool npm run dev",
+        "dev:teamsfx:launch-testtool": "env-cmd --silent -f env/.env.testtool teamsapptester start",
+        "dev": "nodemon --inspect=9239 --signal SIGINT ./src/index.js",
+        "start": "node ./src/index.js",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "watch": "nodemon --exec \"npm run start\""
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com"
+    },
+    "dependencies": {
+        "@azure/data-tables": "^13.2.2",
+        "@azure/identity": "^4.2.0",
+        "@azure/msal-browser": "^3.11.0",
+        "@azure/msal-node": "^2.9.0",
+        "@azure/openai": "^1.0.0-beta.12",
+        "@microsoft/microsoft-graph-client": "^3.0.4",
+        "@microsoft/teams-ai": "^1.1.0",
+        "adaptivecards": "^3.0.4",
+        "azure-storage": "^2.10.7",
+        "botbuilder": "^4.22.2",
+        "botframework-connector": "^4.22.2",
+        "dotenv": "^16.4.5",
+        "e": "^0.2.33",
+        "expres": "^0.0.5",
+        "express": "^4.19.2",
+        "fs": "^0.0.1-security",
+        "http": "^0.0.1-security",
+        "jsonwebtoken": "^9.0.2",
+        "jwt-decode": "^2.2.0",
+        "langchain": "^0.2.7",
+        "luxon": "^3.4.3",
+        "memorystorage": "^0.12.0",
+        "moment-timezone": "^0.5.45",
+        "node-fetch": "^2.6.0",
+        "pem": "^1.14.8",
+        "q": "^1.5.1",
+        "request": "^2.88.2",
+        "restify": "^10.0.0",
```

Comment:
Are we using all of these package?
If not, please remove.

Reply:
Done